web/app/controllers/Application.java

```diff
@@ -93,6 +94,38 @@ public static Result index(String path) {
         return serveAsset("");
     }
 
+    /**
+     * Creates a wrapping ObjectNode containing config information
+     *
+     * @return Http Result instance with app configuration attributes
+     */
+    public static Result appConfig() {
+        ObjectNode response = Json.newObject();
+        Boolean isInternal = Play.application().configuration().getBoolean(LINKEDIN_INTERNAL_KEY, false);
```

Comment:
Great! I'll make that change, it's not expected to be used anywhere except here which is why i made it local variable. Thanks for the feeback